amundsen_application/static/js/components/common/ListItem/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+
+import { SearchIndexParams} from './types';
+import { SearchResult, SearchResultType, TableSearchResult } from "../../../ducks/search/types";
+
+import TableListItem from './TableListItem';
+
+interface ListItemProps {
+  params: SearchIndexParams;
+  item: SearchResult;
+}
+
+
+export default class ListItem extends React.Component<ListItemProps, {}> {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    switch(this.props.item.type) {
```

Comment:
Since this code block is specific to the SearchResults and won't be used anywhere else in the application as a generic `ListItem`, I think we should not put this in common, or maybe rename to `SearchListItem` if you believe that search would be in multiple places in the application.

Reply:
We are also building a bookmark feature. The user profile page will display a list of bookmarks and use the same data format as `SearchResult`. I am trying to figure out a better name that fits both use cases and isn't too confusing or awkward. `Result` or `Item` aren't really descriptive enough, and `SearchResult` is too narrow.

Completely open to suggestions.

Reply:
Maybe `ResourceListItem`.

Reply:
`SearchResult => Resource` and `ListItem => ResourceListItem`? On both the search and bookmark sections we would show `Resource[]`. Sounds like this might work.